### DISCUSSION TITLE
How can I include a font in the executable for a tkinter GUI app?

### DISCUSSION THREAD

**Discussion by ThomasFreedman** [marcelotduarte/cx_Freeze#2042] [Q&A]
If I run this code in the same venv I use to build the app with cx_freeze I see many fonts, including the "Liberation Mono" font my app uses, but it isn't used when I run the binary executable produced by cx_freeze. How can tis be resolved?

```
#!/usr/bin/env python3
from tkinter import Tk, font

root = Tk()
font_tuple = font.families()
root.destroy()
for font in font_tuple:
    print(font)
```

**Comment by marcelotduarte** [comment]
It looks like you found a bug. If you install without the binaries, it works:
`pip install --no-binary=cx_Freeze cx_Freeze`

However, this way the applications you generate will only run on the version of Linux you are using or later.

EDIT:
See #2046
